XiEditor/EditViewController.swift

```diff
@@ -445,7 +445,7 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
     
     // Determines the gesture type based on flags and click count.
     private func clickGestureType(event: NSEvent) -> String {
-        let flags = event.modifierFlags.rawValue >> 16
+        _ = event.modifierFlags.rawValue >> 16
```

Comment:
I think this is a relic of how we used to send modifiers; we can just remove it now.